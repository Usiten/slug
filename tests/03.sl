a = 1;
b = a + 1;
c = (4 * b) + a;
= a;
= b;
= c;