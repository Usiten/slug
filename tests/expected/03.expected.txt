-- Stack Top --
>9
 2
 1
---------------
-- Var Data --
a: 1
b: 2
c: 9
---------------
